examples/benchmarking/M5-forecasting.ipynb

```diff
@@ -0,0 +1,3944 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Splitting into Endogeneous and Exogeneous data
</pre><p>I would move this comment in a markdown cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/7113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>